test/JDBC/input/BABEL-4271-vu-prepare.sql

```diff
@@ -0,0 +1,8 @@
+-- Test to check like escape null and like escape ''
+create table testvikasprj(a varchar(30), b varchar(30));
```

Comment:
We follow the convention of giving table names like babel--vu-prepare-. For this case it can be something like babel-4271-vu-prepare-t1. So please update all the names.

Reply:
updated in the  next revision.